.github/workflows/linux.yml

```diff
@@ -26,11 +26,23 @@ jobs:
           python-version: '3.9'
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade meson PyYAML
           sudo apt update
           sudo apt install -y \
-            doxygen libxcb-xkb-dev valgrind ninja-build \
-            libwayland-dev wayland-protocols bison graphviz
+            doxygen libxcb-xkb-dev valgrind ninja-build meson \
+            libwayland-dev wayland-protocols bison graphviz curl
+          python -m pip install --upgrade PyYAML
+      - name: Install xkeyboard-config
+        run: |
+          # HACK: Force xkeyboard-config version
+          pushd ~
+          XKC_VER=2.39
+          curl -O "https://www.x.org/releases/individual/data/xkeyboard-config/xkeyboard-config-$XKC_VER.tar.xz"
+          tar -xf "xkeyboard-config-$XKC_VER.tar.xz"
+          cd "xkeyboard-config-$XKC_VER"
```

Comment:
mostly a personal preference, but why don't we build from a `git clone --depth=1 --branch=2.39`? tarballs seem like *sooo* last century :)

Reply:
Use master, as per your last comment.